packages/api/src/services/context.servce.spec.ts

```diff
@@ -0,0 +1,116 @@
+import {ContextService} from './context.service';
+
+// Expose protected methods for testing
+class TestContextService extends ContextService {
+  public updateProperty<T>(propertyName: string, value: T): void {
+    this.updateContextProperty(propertyName, value);
+  }
+
+  public getProperty<T>(propertyName: string): T | undefined {
+    return this.getContextPropertyValue(propertyName);
+  }
+
+  public subscribeToProperty<T>(propertyName: string, callback: (value: T | undefined) => void): () => void {
+    return this.subscribe(propertyName, callback);
+  }
+}
+
+describe("ContextService", () => {
```

Comment:
I assume these should be single quotes also?

Reply:
Done